### PR TITLE
[ADD] account_banking_nl_rabo_mt940

### PR DESCRIPTION
**Pull request by hbrunn**
https://code.launchpad.net/~therp-nl/banking-addons/7.0_mt940_structured/+merge/225805

### CODE REVIEW

**Comment by NL66278** [comment]
Approve. Code review and test.

File: account_banking_nl_rabo_mt940/account_banking_nl_rabo_mt940.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            self.current_transaction.remote_owner = (
+                ' '.join(subfields['NAME']))
+
+        if 'ADDR' in subfields:
+            # Do NOT join address fields, array is expected on other code!
+            self.current_transaction.remote_owner_address = subfields['ADDR']
+
+        if 'REMI' in subfields:
+            self.current_transaction.message = '/'.join(
+                filter(lambda x: bool(x), subfields['REMI']))
```
Comment (gurneyalex) [inline account_banking_nl_rabo_mt940/account_banking_nl_rabo_mt940.py]: I'd suggest rewriting these two lines as

```
self.current_transaction.message = '/'.join(sf for sf in subfields['REMI'] if sf)
```

If you'd rather go for `filter()` then you can replace the anonymous function with `None`, which is handled by `filter()` to do precisely this. 

A couple of automated tests could not do any harm.

**Comment by hbrunn** [comment]
Thanks, test will arrive sooner or later (that) in account_banking_mt940, which is where all the magic happens

**Comment by StefanRijnhart** [comment]
Thanks, LGTM. I believe the failed CI is unrelated.

**Comment by hbrunn** [comment]
yes, that's what #38 is supposed to fix

**Comment by StefanRijnhart** [comment]
Merging the target branch instead of rebasing is considered harmful, if I understand correctly from http://arjanvandergaag.nl/blog/clarify-git-history-with-merge-commits.html. Can you rebase?

**Comment by hbrunn** [comment]
yes, better that way

**Comment by StefanRijnhart** [comment]
Thanks! :+1:

**Pull request merged**